docs/articles/nunit/extending-nunit/Engine-Extensibility.md

```diff
@@ -151,3 +151,9 @@ special/myassembly.dll  # include a specific dll in a special directory
 Any assemblies specified in a `.addins` file will be scanned fully, looking for addins and extensions. Any directories specified will be browsed, first looking for any `.addins` files. If one or more files are found, the content of the files will direct all further browsing. If no such file is found, then all `.dll` files in the directory will be scanned, just as if a `.addins` file contained "*.dll."
 
 Assemblies are examined using Cecil, without actually loading them. Info is saved for actual instantiation of extensions on a just-in-time basis.
+
+### Installing Extensions
+
+In [the previous section](#locating-extensions), we note how to use the `.addins` file to utilize an existing extension.
+
+To install an extension, ensure that the binaries will be in a known location, and then create or update a `.addins` file to point to that location so that the assemblies will be scanned and the extensions discovered by NUnit.
```

Comment:
```suggestion
To install an extension, ensure that the binaries will be in a known location, and then create or update a `.addins` file to point to that location so that the assemblies will be scanned and the extensions discovered by NUnit.

Note that in the case of the NUnit Console when installed by NuGet or Chocolatey, extensions installed from the same package manager are detected automatically. The default `.addins` file for these packages contains logic to locate these extensions, and no manual alterations to are required.
```